### ISSUE TITLE
Product Site Information Architecture

### ISSUE THREAD

**Issue by kevinmpowell** [EightShapes/Blocks#149]
* Home (All Products - Featuring Framer and Loader? (name TBD))
 * Framer Landing
    * Getting Started Tutorial (possibly w/Screencast?)
    * Usage Docs
    * Styling Docs
    * Examples

 * Loader Landing
    * Getting Started Tutorial (possibly w/Screencast?)
    * Usage Docs
    * Examples

**Comment by kevinmpowell** [comment]
**ACTION ITEMS**
@nathanacurtis 
* Any reason this Github Pages site shouldn't replace the Wiki? It will be much more seamless to integrate examples and code examples rather than jumping from Wiki (code samples) to Pages site (live examples)
* I'd like to do a quick sketching session with you before starting, agree?
* We'll need to resolve #151 and select a UI framework to build on before starting

**Comment by nathanacurtis** [comment]
Yes, wiki content would migrate to Pages site.
Yes, we will probably need multiple sketching sessions.

On Thu, Jun 11, 2015 at 11:41 AM, Kevin Powell <notifications@github.com>
wrote:

> *ACTION ITEMS*
> @nathanacurtis <https://github.com/nathanacurtis>
>
>    - Any reason this Github Pages site shouldn't replace the Wiki? It
>    will be much more seamless to integrate examples and code examples rather
>    than jumping from Wiki (code samples) to Pages site (live examples)
>    - I'd like to do a quick sketching session with you before starting,
>    agree?
>    - We'll need to resolve #151
>    <https://github.com/EightShapes/Blocks/issues/151> and select a UI
>    framework to build on before starting
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EightShapes/Blocks/issues/149#issuecomment-111177334>.
>



-- 

*Nathan Curtis*
Principal, EightShapes LLC
nathan@eightshapes.com
@nathanacurtis
703-296-8831

**Comment by kevinmpowell** [comment]
## Proposed IA

### Home
* Overview of the "Blocks Toolkit"
* Attention-grabbing use cases "why would I want to use this?"
* Link out to "Who We Are" - eightshapes.com ?
* Differentiate from UI Frameworks (Bootstrap, Foundation) ?
* Quick start guide:
  * Download the JS & CSS and place in your document
  * Anatomy of a "Blocks Project"
    * Show typical directory structure, i.e. `components/` folder, `config.json` etc.
* Primary Links to:
  * Component Loader
  * Page Framer
* Secondary Links to:
  * Docs (config.json)
  * Goodies (bower)
  * Open Source (Github Repo)

### Component Loader
* Overview of what component loader is
* Highlighted use cases (not code examples, just visuals to show why you would use)
* NOTE: For prototyping only
* Links to:
  * Tutorial
  * Component File Documentation
  * Data Injection
  * Config Options
  * Examples (single components + Code)
  * Showcase (full pages/component combinations, no code, client examples?)

### Page Framer
* Overview of what component loader is
* Highlighted use cases (not code examples, just visuals to show why you would use)
* NOTE: Production ready (IE10+, FF, Chrome, Safari)
* Links to:
  * Tutorial
  * Styling
  * Config Options
  * Examples (single instances + Code - possibly Bootstrap, Foundation, etc. themed variants?)
  * Showcase (full page designs using Page Framer, client examples?)

### Docs (config.json)
* Highlights what it is:
  * Global Blocks Toolkit Config
  * Individual Blocks Product Config (Component Loader, Page Framer)
  * Global Data Store

### Goodies
* For the devs
  * bower install option
  * yeoman project config?

**Comment by kevinmpowell** [comment]
@nathanacurtis I propose a name change of "Load" to "Component" - the term "Load" is pretty overloaded in the web world, and all of the products "Load" something. Thoughts?

**Comment by nathanacurtis** [comment]
I'd assert Component is more vaguely used than Load, but agree we should
consider other product names. Can you open a new issue for us to track that
discussion?

On Wed, Jun 17, 2015 at 12:30 PM, Kevin Powell <notifications@github.com>
wrote:

> @nathanacurtis <https://github.com/nathanacurtis> I propose a name change
> of "Load" to "Component" - the term "Load" is pretty overloaded in the web
> world, and all of the products "Load" something. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EightShapes/Blocks/issues/149#issuecomment-112869880>.
>



-- 

*Nathan Curtis*
Principal, EightShapes LLC
nathan@eightshapes.com
@nathanacurtis
703-296-8831

**Comment by kevinmpowell** [comment]
> I'd assert Component is more vaguely used than Load

touché #179 assigned to @nathanacurtis feel free to change assignment

**Comment by kevinmpowell** [comment]
The preliminary IA for this site is complete
